src/main/java/duke/Event.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDateTime start;
+    protected LocalDateTime end;
+
+    public Event(String description, LocalDateTime start, LocalDateTime end) {
+        super(description);
+        this.start = start;
+        this.end = end;
+    }
+
+    @Override
+    public String toString() {
+        DateTimeFormatter customFormatter = DateTimeFormatter.ofPattern("MMM d yyyy h:mma");
+        String formattedDateTime = this.start.format(customFormatter);
+        String formattedDateTime1 = this.end.format(customFormatter);

```

Comment:
Perhaps change `formattedDateTime1` to something more descriptive, like `formattedEndDateTime`